plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -83,14 +91,15 @@ function getCodeLinkInfo(entity: Entity): CodeLinkInfo {
 
 type AboutCardProps = {
   entity: Entity;
+  variant?: string;
```

Comment:
Do we really need this? The AboutCard is only ever used on Overview pages, inside a grid.

Reply:
Not sure. What if it's not desired to set the height suitable to the grid? With this option, you have the possibility to configure it in the app like the other plugin cards, as well.

Reply:
OK, sure.